.github/CODEOWNERS

```
@@ -0,0 +1 @@
+* @sweetpea22 @web3/web3-js
```

```suggestion
* @web3/web3-js
```
@sweetpea22  is not in our org?

---

That's Cindy, added her as outside collaborator on this repo^^